### ISSUE TITLE
Migration not finalized

### ISSUE THREAD

**Issue by AcePandora** [vrchat-community/creator-companion#398]
### Describe the bug

I have been having this bug for a while and I can't seem to figure out how to fix it as it seems to be just me who is having it. every time I try to migrate my projects from 2019.4.31f1 to 2022.3.6f1 in the VRCC I always get "Migration not finalized". I can still run and use the old 2019.4.31f1 for now but the projects just won't migrate even though everything else is up to date. 

### To Reproduce

1. make sure unity and VRCC are updated
2. go to migrate a 2019.4.31f1 to 2022.3.6f1
Unity

### Expected behavior

It will take you to the migrated version of the original folder but say that it failed to finalize it leaving it stuck on 2019.4.31f1

### Screenshots

![IMG_4050](https://github.com/vrchat-community/creator-companion/assets/127035677/256fdbed-c66b-49e6-9f9c-0342b9dc1662)
![IMG_4051](https://github.com/vrchat-community/creator-companion/assets/127035677/a1dda2a2-d0e4-4b32-bd5c-6f5383ba2105)
![IMG_4052](https://github.com/vrchat-community/creator-companion/assets/127035677/b5c54324-bbce-4b4e-a761-73a9bd22cb78)

### Related Product

VCC (VRChat Creator Companion)

### Product Version

2.2.1

### OS

Windows 11

**Comment by momo-the-monster** [comment]
Do your logs have any information about why the migration failed? There are some community packages which are not yet compatible.

**Comment by AcePandora** [comment]
> Do your logs have any information about why the migration failed? There are some community packages which are not yet compatible.

yes, I've looked in the logs and tried closing all community packages to see if that works but it still gives the same thing. So i try to create a new project that is in 2022 but it opens up unity and it refuses to open up the project at all.
